openmetadata-ui/src/main/resources/ui/src/components/Explore/EntitySummaryPanel/EntitySummaryPanel.test.tsx

```diff
@@ -118,6 +119,35 @@ jest.mock('../../../hooks/useEntityRules', () => ({
   })),
 }));
 
+jest.mock('../../../utils/SearchClassBase', () => {
+  const actual = jest.requireActual('../../../utils/SearchClassBase');
+  const originalInstance = actual.default;
+  
+  // Use Object.assign to copy all properties and methods
+  const mockSearchClassBase = Object.assign({}, originalInstance);
+  
+  // Copy prototype methods
+  const prototype = Object.getPrototypeOf(originalInstance);
+  Object.getOwnPropertyNames(prototype).forEach((name) => {
+    if (name !== 'constructor' && typeof prototype[name] === 'function') {
+      mockSearchClassBase[name] = prototype[name].bind(originalInstance);
+    }
+  });
+  
+  // Override getEntityLink to handle null/undefined gracefully
+  mockSearchClassBase.getEntityLink = jest.fn().mockImplementation((entity) => {
+    if (!entity || !entity.entityType) {
+      return '';
+    }
+    return originalInstance.getEntityLink(entity);
+  });
+  
```

Comment:
Why do we need this complex logic for mocking? Can't we simply return a string as a link?